docs/GettingStarted.md

```diff
@@ -81,31 +81,65 @@ jest --init
 
 ### Using Babel
 
-To use [Babel](http://babeljs.io/), install the `babel-jest` and `@babel/core` packages via `yarn`:
+To use [Babel](http://babeljs.io/), install required dependencies via `yarn`:
 
 ```bash
-yarn add --dev babel-jest @babel/core
+yarn add --dev babel-jest @babel/core @babel/preset-env @babel/preset-react
 ```
 
-Don't forget to add a [`babel.config.js`](https://babeljs.io/docs/en/config-files) file in your project's root folder. For example, if you are using ES2018 and [React](https://reactjs.org) with the [`@babel/preset-env`](https://babeljs.io/docs/en/babel-preset-env) and [`@babel/preset-react`](https://babeljs.io/docs/en/babel-preset-react) presets:
+Configure Babel to target your current version of Node by creating a `babel.config.js` file in the root of your project:
 
-```js
+```javascript
+// babel.config.js
 module.exports = {
-  presets: ['@babel/preset-env', '@babel/preset-react'],
+  presets: [
+    [
+      '@babel/preset-env',
+      {
+        targets: {
+          node: 'current',
+        },
+      },
+    ],
+    '@babel/preset-react',
+  ],
 };
 ```
 
-You are now set up to use all ES2018 features and React-specific syntax.
+If you are not using React you can exclude `@babel/preset-react` from the above commands/config.
+
+The ideal configuration for Babel will depend on your project. See [Babel's docs](https://babeljs.io/docs/en/babel-preset-env) for more details.
+
+Jest will set `process.env.NODE_ENV` to 'test' if it's not set to something else, you can use that in your configuration to conditionally setup only the compilation needed for Jest, e.g.
+
+```javascript
+// babel.config.js
+module.exports =
```

Comment:
Instead of a ternary on `process.env.NODE_ENV` directly, I'd do something like this:

```js
module.exports = api => {
  const isTest = api.env('test');
  api.cache(true);

  return ...
};
```

See https://babeljs.io/docs/en/config-files#apienv

---

Or even check `api.caller(caller => caller && caller.name === 'babel-jest')` if we just wanna check if it's Jest.

Reply:
Done. Although I'm not sure that usage of `api.cache` is correct, so I left it out. I think the caller check is probably unnecessary.